### PR TITLE
Add skierowanie command

### PR DESCRIPTION
**Pull request by AdamSiekierski**
I've created a `!skierowanie` command for telling people, they'll need to work for a bit longer on fundamentals.

It sends an embed and some useful links for selected category

*Example*
image

There aren't many links yet, but later we can add more :D

Take the look at the code and way the command looks like, a share your thoughts ;)

### CODE REVIEW

File: src/commands/skierowanie.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    const categoryFilter = args[1]?.toLowerCase();
+    const linksFiltered = links.filter(
+      ({ category }) => !(category && categoryFilter && category !== categoryFilter)
+    );
+
+    const linksMessage = 'Z powyższym skierowaniem należy udać się na poniższe strony internetowe:';
+
+    await msg.channel.send(skierowanieEmbed);
+    return msg.channel.send([linksMessage, ...linksFiltered.map((link) => link.address)]);
```
Comment (mmiszy) [inline src/commands/skierowanie.ts:56]: A to się nie może razem wysłać?

File: src/commands/skierowanie.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      .setTimestamp()
+      .setFooter(
+        'Type of Web, Discord, Polska',
+        'https://cdn.discordapp.com/avatars/574682557988470825/6b0fab28093e6020f497fda41bdd3219.png?size=64'
+      );
+
+    const categoryFilter = args[1]?.toLowerCase();
+    const linksFiltered = links.filter(
+      ({ category }) => !(category && categoryFilter && category !== categoryFilter)
+    );
```
Comment (mmiszy) [inline src/commands/skierowanie.ts:51]:
```suggestion
    const linksFiltered = categoryFilter ? links.filter(
      ({ category }) => category === categoryFilter
    ) : links;
```

**Review by mmiszy — CHANGES_REQUESTED** [review]

File: src/commands/skierowanie.ts (hunk 1 above)
Comment (AdamSiekierski) [reply src/commands/skierowanie.ts:56]: Nie może, embedy muszą być w osobnych wiadomościach

File: src/commands/skierowanie.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import Discord from 'discord.js';
+import { Command } from '../types';
+
+type Link = {
```
[the remark this answers is not in the document]
Comment (AdamSiekierski) [reply src/commands/skierowanie.ts:4]: Fakt

**Review by mmiszy — APPROVED** [review]

**Comment by mmiszy** [comment]
@all-contributors please add @AdamSiekierski for code

**Pull request merged**